test_data/yast/select_modules_and_patterns.yaml

```diff
@@ -0,0 +1,11 @@
+---
+software:
+  patterns:
```

Comment:
Are you going to adjust `select_patterns` module to also select patterns based on test_data? I wonder if we miss settings in the test suite yamls, as it still states `PATTERNS: all`

Reply:
yes, in that VR http://waaa-amazing.suse.cz/tests/13692# we get PATTERNS=all in json, despite having PATTERNS=default,yast-development in the settings tab. Looks like some magic is done at some point, Trying to figure out where.

Reply:
Then also validate_software fails, I also need to find why

Reply:
It's not magic, we parse yaml during the runtime, so it overwrites settings you have provided elsewhere. But I still do not understand, why not simply set them here as you did manually in your VRs.

Reply:
It is not set in YAML. This is only for validate_software

Reply:
And well, that's what I call "magic", it's quite annoying to have some piece of code modifying what we set. then why to set any settings at all, if they get overwritten ?

Reply:
![image](https://user-images.githubusercontent.com/28305643/99967610-66ecb580-2d98-11eb-8e92-91d47adbd9f0.png)
It is set in yaml

Reply:
No, you were actually right, I was looking at something else and didn't even look the vars section... I have to remove that.

Reply:
In that case, I thought it made sense to put it in the test suite as it applies to all variants of the test suite. But then I got confused myself as I forgot it was also set here WDYT ? is it better to set it in the YAML schedule or in the jobgroup ?

Reply:
So in conclusion, I think in this case the best place to set it is the test suite, as they will all have the same values for PATTERNS